eng/pipelines/aggregate-reports.yml

```diff
@@ -27,7 +32,7 @@ jobs:
       displayName: 'Build All Libraries'
       inputs:
         mavenPomFile: pom.xml
-        options: '$(DefaultOptions) -T 1C -DskipTests -Dgpg.skip -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Djacoco.skip=true -Drevapi.skip=true -Paggregatereports'
+        options: '$(DefaultOptions) -T 2C -DskipTests -Dgpg.skip -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Djacoco.skip=true -Drevapi.skip=true -Paggregatereports'
```

Comment:
Why 2C?

Reply:
During installation and building libraries it has shown to be more performant than 1C. I'd imagine this is due to processing being bound on I/O and not CPU.